src/rules/selector-pseudo-class-parentheses-space-inside/index.js

```diff
@@ -0,0 +1,86 @@
+import selectorParser from "postcss-selector-parser"
+import {
+  report,
+  ruleMessages,
+  styleSearch,
+  validateOptions,
+} from "../../utils"
+
+export const ruleName = "selector-pseudo-class-parentheses-space-inside"
+
+export const messages = ruleMessages(ruleName, {
+  expectedOpening: "Expected single space after \"(\"",
+  rejectedOpening: "Unexpected whitespace after \"(\"",
+  expectedClosing: "Expected single space before \")\"",
+  rejectedClosing: "Unexpected whitespace before \")\"",
+})
+
+export default function (expectation) {
+  return (root, result) => {
+    const validOptions = validateOptions(result, ruleName, {
+      actual: expectation,
+      possible: [
+        "always",
+        "never",
+      ],
+    })
+    if (!validOptions) { return }
+
+    root.walkRules(rule => {
+      selectorParser(selectorTree => {
+        selectorTree.eachPseudo(pseudoNode => {
+          if (pseudoNode.parent
+            && pseudoNode.parent.parent
+            && pseudoNode.parent.parent.type === "pseudo"
```

Comment:
Can be simplified with `_.get()`: `if (_.get(pseudoNode, 'parent.parent.type) === "pseudo")`